src/develop/masks/masks.c

```diff
@@ -609,6 +603,23 @@ void dt_masks_gui_form_save_creation(dt_develop_t *dev, dt_iop_module_t *module,
   if(gui) dev->form_gui->formid = form->formid;
 }
 
+static void _dt_masks_group_set_name_from_module(dt_iop_module_t *module, dt_masks_form_t *grp)
+{
+  gchar *module_label = dt_history_item_get_name(module);
+  snprintf(grp->name, sizeof(grp->name), "grp %s", module_label);
+  g_free(module_label);
+}
+
+dt_masks_form_t *dt_masks_group_create(dt_iop_module_t *module, dt_masks_type_t type)
```

Comment:
set this one static and remove leading `dt_masks_` from name to just have `_group_create`